installer/manifests/keptn/charts/control-plane/templates/core.yaml

```
@@ -264,9 +288,26 @@ spec:
             # Session cookie timeout in minutes
             - name: SESSION_TIMEOUT_MIN
               value: "{{ .Values.bridge.oauth.sessionTimeoutMin}}"
+            # Timeout of validation data (state, nonce, code verifier) during login
+            - name: SESSION_VALIDATING_TIMEOUT_MIN
+              value: "{{ .Values.bridge.oauth.sessionValidatingTimeoutMin}}"
             # Correlates to trust proxy number of hops as defined at http://expressjs.com/en/guide/behind-proxies.html
             - name: TRUST_PROXY
               value: "{{ .Values.bridge.oauth.trustProxy }}"
+            - name: MONGODB_HOST
+              value: keptn-mongo:27017
+            - name: MONGODB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: mongodb-credentials
+                  key: mongodb-user
+            - name: MONGODB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mongodb-credentials
+                  key: mongodb-password
+            - name: MONGODB_DATABASE
+              value: openid
```

I would make this configurable with an optional. Please, also add this inside the `value.yaml` file around line 9.
```suggestion
              value: {{ .Values.mongo.auth.bridgeAuthDatabase | default "openid" }}
```